src/app.ts

```diff
@@ -1,8 +1,10 @@
 import Koa, { Context } from 'koa';
+import logger from 'koa-logger';
 import { constants as statusCodes } from 'http2';
 
 const app = new Koa();
 
+app.use(logger());
```

Comment:
Probably want a way to disable this for the test suite.

Reply:
https://github.com/libero/article-store/pull/7/commits/cec623c88f9ec5b39430616202be5a8de3b719c3 works (for now).